scripts/gui-entity.lua

```diff
@@ -252,7 +252,8 @@ local function add_creative_chest_group_number_and_update_gui(creative_chest_ent
 				chest_groups,
 				group_number,
 				index_in_group,
-				new_group_number
+				new_group_number,
+                creative_chest_entity
```

Comment:
Formatting?

Reply:
Tabs vs spaces issue, changed to tabs to match the rest.